docs/index.md

```diff
@@ -32,6 +37,18 @@ resource "mcs_kubernetes_cluster" "mycluster"{
 }
 ```
 
+## Authentication
+
+The MCS provider supports username/password authentication. Preconfigured provider file with username and project_id can be download on https://mcs.mail.ru/app/ portal. Go to Containers -> Kubernetes clusters -> click the cluster name -> "MCS Terraform Provider" -> click on the "Download MCS provider file".
```

Comment:
use can use direct links via [link](http://link)

Reply:
use this path for mcs' provider file 
https://mcs.mail.ru/app/mcs5615275815/project/terraform/

Reply:
use `` for variables to make it visible. example `project_id`